Apps/W1/Shopify/app/src/Invoicing/Codeunits/ShpfyPostedInvoiceExport.Codeunit.al

```diff
@@ -267,7 +247,7 @@ codeunit 30316 "Shpfy Posted Invoice Export"
     var
         ShpfyCustomer: Record "Shpfy Customer";
     begin
-        TempShpfyOrderHeader."Bill-to Name" := SalesInvoiceHeader."Bill-to Name";
+        TempShpfyOrderHeader."Bill-to Name" := Format(SalesInvoiceHeader."Bill-to Name");
```

Comment:
Does format actually strip the text, or does it only silence the warning? It seems dangerous...

Reply:
Only for the warning

Reply:
That's not good. Than it will be a run time error if the name is too big. It should be a CopyString if we actually can have a longer name on one side.

Reply:
@coderabbitai pause